apps/src/netsim/NetSimStatusPanel.html.ejs

```diff
@@ -1,18 +1,20 @@
 <%
 var i18n = require('../../locale/current/netsim');
 %>
-<% if (remoteNodeName) { %>
-<p>Connected to <%= remoteNodeName %></p>
-<% } %>
+<div class="content-wrap">
+  <% if (remoteNodeName) { %>
+  <p>Connected to <%= remoteNodeName %></p>
+  <% } %>
```

Comment:
The outer `<div class="content-wrap">` the only new thing here.  The status panel used to have one-off styling that provided its padding, so it was behaving inconsistently from the other panels.